CI fails due to checkstyle error

**Describe the bug**
<!-- Please explain what happened and provide the context in which the bug occurred. -->
```
Error: [ERROR] src/main/java/com/teragrep/nlf_01/types/AuditLogsCLType.java:[143,9] (blocks) NeedBraces: 'if' construct must use '{}'s.
```

**Expected behavior**
<!-- Please tell us why you think the behavior is unexpected. If you can, please copy-paste logs or error messages you got while facing the bug. -->
build passes

**How to reproduce**
<!-- Please provide us reproducible step-by-step guide on how to reproduce the bug. Please keep in mind that non-reproducible issues will be closed. -->
run CI with current commit

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

**Software version**
<!-- e.g. 3.1.4 -->

**Desktop (please complete the following information if relevant):**
 - OS: <!-- [e.g. iOS] -->
 - Browser: <!-- [e.g. chrome, safari] -->
 - Version: <!-- [e.g. 22] -->

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/src/main/java/com/teragrep/nlf_01/types/AuditLogsCLType.java
+++ b/src/main/java/com/teragrep/nlf_01/types/AuditLogsCLType.java
@@ -140,8 +140,9 @@ public final class AuditLogsCLType implements EventType {
 
     @Override
     public boolean equals(final Object o) {
-        if (o == null || getClass() != o.getClass())
+        if (o == null || getClass() != o.getClass()) {
             return false;
+        }
         final AuditLogsCLType that = (AuditLogsCLType) o;
         return Objects.equals(parsedEvent, that.parsedEvent) && Objects.equals(realHostname, that.realHostname)
                 && Objects.equals(componentNameForPartitions, that.componentNameForPartitions);

